Components/Utils/ModelBuilder.cs

```diff
@@ -109,16 +109,16 @@ public AppTop BuildAppTop()
                 return new AppTop
                 {
                     AppName = new List<Link> { _model.ApplicationTitle },
-                    SignIn = _model.Builder.BuildHideableHrefOnlyLink(_model.Settings.SignInLinkUrl, _model.ShowSignInLink),
-                    SignOut = _model.Builder.BuildHideableHrefOnlyLink(_model.Settings.SignOutLinkUrl, _model.ShowSignOutLink),
+                    SignIn = BuildHideableHrefOnlyLink(_model.Settings.SignInLinkUrl, _model.ShowSignInLink),
```

Comment:
Just to confirm: ALL removals of  `_model.Builder` is because these functions are ALL static, right?

Reply:
Yes that is correct